Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+    <PropertyGroup>
+        <PackageProjectUrl>https://zaid-ajaj.github.io/Feliz/</PackageProjectUrl>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Nuget and other views have better support for these License Expressions rather than a raw License File.